src/ToolBox/SOS/Strike/util.cpp

```diff
@@ -4138,7 +4155,40 @@ void ResetGlobals(void)
 
 HRESULT LoadClrDebugDll(void)
 {
-#ifndef FEATURE_PAL
+#ifdef FEATURE_PAL
+    if (!g_palInitialized)
+    {
+        int err = PAL_Initialize(0, NULL);
+        if(err != 0)
+        {
+            return E_FAIL;
+        }
+        g_palInitialized = TRUE;
+    }
+    if (g_clrData == NULL)
+    {
+        HMODULE hdac = LoadLibraryA("/ssd/coreclr/binaries/Product/amd64/debug/" MAKEDLLNAME_A("mscordaccore"));
```

Comment:
You don't need to do the initialization protection. The PAL_Initialize is safe to be called multiple times.